doc/configuration.rst

```diff
@@ -1746,9 +1758,19 @@ Building without executing examples
 Sphinx-Gallery can parse all your examples and build the gallery
 without executing any of the scripts. This is just for speed
 visualization processes of the gallery and the size it takes your
-website to display, or any use you can imagine for it. To achieve this you
-need to pass the no plot option in the build process by modifying
-your ``Makefile`` with:
+website to display, or any use you can imagine for it.
+
+This can be done by setting the ``plot_gallery`` configuration in the
```

Comment:
This was a more opinionated change, to talk about the makefile second, but I thought it was more consistent with the rest of the docs which generally talk about setting the conf in the `conf.py` file first